postreise/analyze/generation/summarize.py

```diff
@@ -129,3 +133,42 @@ def summarize_hist_gen(hist_gen_raw: pd.DataFrame, all_resources: list) -> pd.Da
     result.rename(columns=type2label, inplace=True)
 
     return result
+
+
+def get_generation_time_series_by_resources(scenario, area, resources, area_type=None):
+    """Get time series generation for each resource in certain area of a scenario
+
+    :param powersimdata.scenario.scenario.Scenario scenario: scenario instance
+    :param str area: one of *loadzone*, *state*, *state abbreviation*,
+        *interconnect*, *'all'*
+    :param str/list resources: one or a list of resources
+    :param str area_type: one of *'loadzone'*, *'state'*, *'state_abbr'*,
+        *'interconnect'*
+    :return: (*pandas.DataFrame*) -- data frame of time series generation for each
```

Comment:
data frame of time series generation for --> times series of generation for

Reply:
Done.